lib/test/Test/Pos/Helpers.hs

```diff
@@ -1,18 +1,10 @@
 {-# LANGUAGE AllowAmbiguousTypes #-}
 {-# LANGUAGE RankNTypes          #-}
 
-module Test.Pos.Util
-       ( HasStaticConfigurations
-       , withDefConfiguration
-       , withDefInfraConfiguration
-       , withDefNodeConfiguration
-       , withDefGtConfiguration
-       , withDefUpdateConfiguration
-       , withDefConfigurations
-       , withStaticConfigurations
-
```

Comment:
Wasn't it easier to just move the whole file to `lib`? What is the rationale behind splitting test utils into two files (other than "let's move only things that are _necessary_ for wallet")?

Reply:
there is at least one reason not to move the whole file: 
`import           Test.Hspec`,
so I wouldn't like `lib` depends on `hspec` package